datahub-web-react/src/app/entity/user/UserGroups.tsx

```diff
@@ -88,9 +88,9 @@ export default function UserGroups({ urn, initialRelationships, pageSize }: Prop
         getGroups({ variables: { urn, start, count: pageSize } });
     };
 
-    const relationships = groupsData ? groupsData.corpUser?.relationships : initialRelationships;
-    const total = relationships?.total || 0;
-    const userGroups = relationships?.relationships?.map((rel) => rel.entity as CorpGroup) || [];
+    const relationships = groupsData ? groupsData.corpUser?.relationships?.relationships : initialRelationships;
```

Comment:
Are we doing this just for cleanup?

Reply:
actually no. Because we now have both roles and groups in relationships, we have to filter

Reply:
ah. a bit confusing. we could probably just split into multiple relationship queries since they are usedin different scenarios

Reply:
Done :)